asteroid/engine/dataloader.py

```diff
@@ -0,0 +1,33 @@
+import torch
+from asteroid.data.librimix_dataset import LibriMix
+import os
+from torch.utils.data import DataLoader
+import torch.nn.functional as F
+
+
+def collate_fn(batch):
+
+    inputs = []
+    targets = []
+    sizes = []
+    if batch is not None:
+        max_len = 0
+        for elm in batch:
+            mixture, sources = elm
+            if len(mixture) > max_len:
+                max_len = len(mixture)
```

Comment:
```suggestion
        max_len = max(map(lambda x: len(x[0]), batch))
```

Reply:
Or `max(len(x[0] for x in batch))` which I think is a bit more readable

Reply:
Yes, true ^^